examples/eks-cluster-with-observability/opensearch.tf

```diff
@@ -0,0 +1,74 @@
+resource "aws_elasticsearch_domain" "opensearch" {
+  domain_name           = "opensearch"
+  elasticsearch_version = "OpenSearch_1.1"
+
+  cluster_config {
+    instance_type          = "m4.large.elasticsearch"
+    instance_count         = 3
+    zone_awareness_enabled = true
+    zone_awareness_config {
+      availability_zone_count = 3
+    }
+  }
+  node_to_node_encryption {
+    enabled = true
+  }
+  domain_endpoint_options {
+    enforce_https       = true
+    tls_security_policy = "Policy-Min-TLS-1-2-2019-07"
+  }
+  encrypt_at_rest {
+    enabled = true
+  }
+  ebs_options {
+    ebs_enabled = true
+    volume_size = var.ebs_volume_size
+  }
+
+  advanced_security_options {
+    enabled                        = true
+    internal_user_database_enabled = true
+    master_user_options {
+      master_user_name     = var.opensearch_dashboard_user
+      master_user_password = var.opensearch_dashboard_pw
+    }
+  }
+}
+
+variable "opensearch_dashboard_user" {
+  type = string
+}
+
+variable "opensearch_dashboard_pw" {
+  type      = string
+  sensitive = true
+}
+
+variable "ebs_volume_size" {
+  type        = number
+  description = "volume size in gigabytes"
+}
+
+resource "aws_iam_policy" "fluentbit-opensearch-access" {
+  name        = "fluentbit-opensearch-access"
+  description = "IAM policy to allow Fluentbit access to OpenSearch"
+  policy      = data.aws_iam_policy_document.fluentbit-opensearch-access.json
+}
+
+resource "aws_elasticsearch_domain_policy" "opensearch_policy" {
+  domain_name = aws_elasticsearch_domain.opensearch.domain_name
+
+  access_policies = <<POLICIES
```

Comment:
Use `data` block for defining policy statements.

Reply:
Whoops, thanks! Fixed.